src/interactions/moderation/anti-raid-nuke.ts

```diff
@@ -2,48 +2,164 @@ import { ApplicationCommandOptionType } from 'discord-api-types/v10';
 
 export const AntiRaidNukeCommand = {
 	name: 'anti-raid-nuke',
-	description: 'Bans all members that have joined recently, with new accounts',
+	description: 'Handle raids and nukes',
 	options: [
 		{
-			name: 'join',
-			description: 'How old should a member be for the cybernuke to ignore them (server join date)?',
-			type: ApplicationCommandOptionType.String,
-			required: true,
-		},
-		{
-			name: 'age',
-			description: "How old should a member's account be for the cybernuke to ignore them (account age)?",
-			type: ApplicationCommandOptionType.String,
-			required: true,
-		},
-		{
-			name: 'pattern',
-			description: "A pattern the member's username should match (case insensitive, full match)",
-			type: ApplicationCommandOptionType.String,
-		},
-		{
-			name: 'avatar',
-			description: 'The avatar hash the user or member should have (exact match)',
-			type: ApplicationCommandOptionType.String,
+			name: 'manual',
+			description: 'Manually filters and bans members based on various criterias',
```

Comment:
I don't think "manual" is a good choice of wording here.
After all, banning based on filters is the exact opposite of a manual approach.
I feel like "filters" or similar would be semantically preferable